tests/frontend/integration/test_integrations.py

```diff
@@ -35,3 +35,31 @@ def test_summary_visual_browse(self):
 
     def test_user_permissions(self):
         tests.user_permissions(self.driver, self.live_server_url)
+
+    # tests for /lit/assessment/2/references/ unworking
```

Comment:
not sure what this means - remove comment or describe issue?

Reply:
I'm still thinking on this one, this seems like some sort of race condition(I think, even though I've just introduced a 5 second wait in one of my tests) with event binding and React that Selenium isn't handling well, I've tried clicking with Helium, Selenium, and injecting raw javascript into the browser. So I'm thinking through next steps here.

Reply:
This ended up being a simple typo, that I had apparently been overlooking the whole time, Nice.